### ISSUE TITLE
Use localized date format in Export-Excel

### ISSUE THREAD

**Issue by jhoneill** [dfinke/ImportExcel#52]
there are two places in Export-excel (line 257 and 297) where US dates are hard coded. I have replace the "m/d/yy h:mm" in mine so that it uses local format. 

   switch ($TargetData.$Name) {
                    {$_ -is [datetime]} {$targetCell.Style.Numberformat.Format = [cultureinfo]::CurrentCulture.DateTimeFormat.ShortDatePattern   }
                }

**Comment by Penkethboy** [comment]
version 4.0.13

I noticed that line 447 of Export-Excel.ps1 also has 'm/d/yy h:mm'

rather than the suggested fix by jhoneill to make it culture dependant

**Comment by jhoneill** [comment]
Hi @Penkethboy  
I went through the file and added some comments for this because there are some special cases where the underlying library recognises the format written in US style and stores it (in this case) as "format 14" which is "Short date".   'm/d/yy h:mm' is captured, but 'm/d/yy' isn't 'mm/dd/yy h:mm' isn't either.  If the 

The way I proposed originally would be great until I shared my file with someone who had a different culture set - then they would get a sheet with my local culture hard coded into it.
